### ISSUE TITLE
only 16 frames of each video

### ISSUE THREAD

**Issue by ramnamaqsood** [dipakkr/3d-cnn-action-recognition#6]
If i am not wrong then in your code you are taking only first 16 frames of each video and ignore remaining....Is there any reason for this?

**Comment by dipakkr** [comment]
Yes. It was because I didn't have a high power GPU. So, that's why I divided only took a certain number of frames from a video ( Frames were chosen using an algorithm not randomly). 

_Randomly choosing 16 frames won't going to work, as it may take first 16frames or last 16 frames. In that frames cannot determine your action properly._

**Comment by dipakkr** [comment]
@ramnamaqsood I hope that answers your question.

**Comment by ramnamaqsood** [comment]
Thanks for your response and could you please tell me that which algorithm you use during frame extraction because i have same issue regarding computational resources for my own video dataset.

**Comment by dipakkr** [comment]
@ramnamaqsood you can check in frame_script.py file.
